windowsforms/Conway's Game-of-Life/VB/readme.md

```diff
@@ -0,0 +1,54 @@
+---
+languages:
+- vb
+products:
+- dotnet-core
+- windows
+page_type: sample
+name: ".NET Core Windows Forms Conway's Game Of Life (Visual Basic)"
+urlFragment: "ConwaysGameOfLife-vb"
```

Comment:
A small change, just to be consistent.

```suggestion
urlFragment: "conways-game-of-life-vb"
```

Reply:
I have replaced spaces with - everywhere.